common/components/BalanceSidebar/AddCustomTokenForm.jsx

```diff
@@ -0,0 +1,115 @@
+// @flow
```

Comment:
This whole file is marked as new, and looks to be a duplicate of `common/components/BalanceSidebar/TokenBalances/AddCustomTokenForm.jsx`